xtask/src/license_headers_check.rs

```diff
@@ -324,8 +324,8 @@ impl CargoDependency {
             toml_edit::Value::InlineTable(table) => {
                 if let (Some(path), Some(version)) = (table.get("path"), table.get("version")) {
                     Some(Self::Full {
-                        path: path.as_str().unwrap_or_default().clone().into(),
-                        version: version.as_str().unwrap_or_default().clone().into(),
+                        path: *(path.as_str().unwrap_or_default()).clone().into(),
```

Comment:
I don’t quite understand this. I can see how a &str could be cloned and into() taking care of allocating a new string. But the patch looks like it derefs the &str and the tries the clone? How would that work? What am I missing?:-)

Reply:
You are right... I was too mechanical there, fixing the clippy warning!